upgrade/scripts/upgrade/util/update-customizations.sh

```diff
@@ -136,6 +136,9 @@ fi
 # Kube-prometheus-stack
 sed -i 's/prometheus-operator/kube-prometheus-stack/g' "$c"
 
+# When upgrading to CSM 1.5 or later, ensure that we remove obsolete cray-service.postgreSql entries (CASMPET-6584).
+yq d -i "$c" 'spec.kubernetes.services.*.cray-service.sqlCluster'
```

Comment:
Same comment as yesterday on Ryan's PR.

https://github.com/Cray-HPE/docs-csm/pull/3805/files#r1198206006

```suggestion
yq4 -i 'del(.'spec.kubernetes.services.*.cray-service.sqlCluster)' "$c"
```

I haven't used the wildcard with the new `del()` function, Ryan's PR was just targeting one key when I made the original recommendation.

Reply:
More information: https://mikefarah.gitbook.io/yq/upgrading-from-v3

Reply:
Yes, that wildcard will work with yq4. I didn't realize that yq4 was on the system earlier when I submitted this. I'll make that change.

Reply:
It's been available but not well advertised.
Recently we're taking advocating for using it for new `yq` usages to make the slide to `yq4` easier.

Reply:
That makes sense. I've pushed a new commit that seems more appropriate based on my review of customizations on Drax (where this issue was discovered). This was tested on Drax and works as expected to update the customizations.